plasmapy/plasma/tests/test_grids.py

```diff
@@ -556,5 +970,22 @@ def debug_volume_avg_interpolator():
     plt.xlim(-11, 11)
 
 
-if __name__ == "__main__":
-    test_volume_averaged_interpolator_known_solutions()
+def test_fast_nearest_neighbor_interpolate():
+    """
+    Confirms that the fast linear interpolator function is equivalent to
+    the np.argmin(np.abs(x-y)) search method for ordered arrays
+    """
+    ax = 100 * np.linspace(0, 1, num=100)
+    pos = np.random.random([300])
```

Comment:
Could you add a random seed to this and to other places where random number generators are used?  In general it's better to have tests that are deterministic, since it's hard to tell when an intermittently failing test is actually fixed.  Thanks!

Reply:
Oh, yes, good idea. Done.

Reply:
Thanks!